packages/logger/test/options.test.ts

```diff
@@ -45,3 +45,12 @@ test("don't mutate default options", (t) => {
   // Ensure the defaults objects remains unchanged
   t.deepEqual(defaultCopy, defaults);
 });
+
+test('json should be set to true if the env var is prsent', (t) => {
+  process.env.OPENFN_LOG_JSON = 'true';
+
+  const opts = calculateOptions({});
+  t.true(opts.json);
+
+  delete process.env.OPENFN_LOG_JSON;
+});
```

Comment:
Should this not be in the CLI? I've been thinking of `logger` strictly being a dependency.

Reply:
Yes, you're absolutely right